ci/cmake.yml

```diff
@@ -0,0 +1,22 @@
+name: CMake CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: create-build-dir
+      run: cmake -E make_directory ${{runner.workspace}}/build
+    - name: cmake-configure
+      working-directory: ${{runner.workspace}}/build
+      run: cmake $GITHUB_WORKSPACE
+    - name: build
```

Comment:
Let's capitalize this, eg `Build`.

Reply:
https://commons.wikimedia.org/wiki/File:Cmake.svg